content/auth/token.textile

```diff
@@ -174,6 +188,20 @@ const rest = new Ably.Rest({
 });
 ```
 
+```[rest_python]
+rest = AblyRest(auth_callback=get_token_request)
+
+async def get_token_request(*args, **kwargs):
+    tokenRequestResponse = await createTokenRequest() // Make a network request to your server
```

Comment:
some camelCase still left over in this example

Reply:
https://github.com/ably/docs/pull/2129/commits/21aef65fb1cce21ebdf652933efaca1247aaf3cb